apps/docs/content/components/Heading/index.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Heading
+description: Use the Heading component to render title or subtitle text
+---
+
+import CustomVideoPlayer from '../../../src/components/custom-video-player'
+import ComponentLayout from '../../../src/layouts/component-layout'
+export default ComponentLayout
+
+![An image displaying each header from h1 to h6.](https://github.com/primer/brand/assets/912236/daf645d9-2ce5-4aba-9bc0-7c95b401c6c8)
+
+## Usage
+
+Headings are used to structure the content on the page, with each heading representing a different level of importance. It is recommended to use headings in a logical manner to create a clear hierarchy and improve accessibility.
+
+- Heading should be used to communicate page organization and hierarchy.
+- Use heading as the title that introduces a page, a section or an article.
+- The heading component will render an html h2 tag without any default styling.
```

Comment:
Think we should capitalize this as it's referring to a component. Also not sure what 'default styling' means here, because we do technically apply some styling. Some readers may interpret this to mean it's browser default.

```suggestion
- The Heading component will render an HTML `h2` tag by default.
```

Reply:
We don't capitalize component names in https://primer.style/design/ , so maybe we shouldn't here either? Or if we disagree with that decision, we could ask @emilybrick about changing component names to PascalCase.

Reply:
I think @mperrotti is right, according to this adr : https://github.com/github/primer/blob/main/adrs/2022-02-15-spelling-of-component-names.md, we use sentence case for design guidelines or UI patterns. 

I'll create a follow-up PR to use sentence case of the Design guidelines released these days.